### ISSUE TITLE
Strange behaviour after screen orientation changes

### ISSUE THREAD

**Issue by sebastianbrenner** [Temzasse/react-modal-sheet#136]
The sheet component works perfectly fine until i change the screen orientation one or multiple times. After changing the orientation this happens:

https://github.com/Temzasse/react-modal-sheet/assets/38284837/2b4fe597-b867-4d00-ada2-01223fd2be7a

This is my implementation of the sheet:

```
<Sheet
   isOpen
   ref={sheetRef}
   onClose={() => snapTo(1)}
   detent="content-height"
   initialSnap={1}
   snapPoints={[300, 72]}
   mountPoint={indexStore?.shadowHost?.shadowRoot?.querySelector('body')!}
   style={{ marginBottom: '56px' }}
>
   <Sheet.Container>
      <Sheet.Header />
      <Sheet.Content>
         <Stack alignContent="center" spacing={2} direction="column" p={2} pt={0}>
            <Typography>Fancy Sheet Content</Typography>
            <TextField
               label="Input"
               type="number"
               size="small"
            >
            <TextField
               label="Input"
               type="number"
               size="small"
            >
            <TextField
               label="Input"
               type="number"
               size="small"
            >
            <Button variant="contained" size="small" sx={{ ml: 'auto' }}>Do something</Button>
         </Stack>
      </Sheet.Content>
   </Sheet.Container>
</Sheet>
```

I'm not quiet sure whether this is a problem with my implementation or a problem with the sheet.

**Comment by Temzasse** [comment]
This is a known issue related to the change of dimensions when the orientation changes. I have not yet found a solution for this as it's a bit unclear whether the bug is in this library or inside framer-motion 🙁

**Comment by Temzasse** [comment]
I think I have **FINALLY** found a solution to this problem! 🎉 😄 

There are a few open issues in framer-motion repo about this same issue: https://github.com/framer/motion/issues/1827 and https://github.com/framer/motion/issues/1659.

Based on the discussion there and after looking into the internals of framer-motion I think I found a way to [fix](https://github.com/Temzasse/react-modal-sheet/commit/ee81930dcec12440ee4b9492c2bb9cd1c7563dd4) the problem where dragging gets messed up after window size is changed.

The fix is released in [v2.2.0](https://github.com/Temzasse/react-modal-sheet/releases/tag/v2.2.0). Please let me know if the problem still persists 😊

**Comment by sebastianbrenner** [comment]
@Temzasse It works perfectly fine now! Thank you very much!

**Comment by Jamie-VisualiseInfo** [comment]
> I think I have **FINALLY** found a solution to this problem! 🎉 😄
> 
> There are a few open issues in framer-motion repo about this same issue: [motiondivision/motion#1827](https://github.com/motiondivision/motion/issues/1827) and [motiondivision/motion#1659](https://github.com/motiondivision/motion/issues/1659).
> 
> Based on the discussion there and after looking into the internals of framer-motion I think I found a way to [fix](https://github.com/Temzasse/react-modal-sheet/commit/ee81930dcec12440ee4b9492c2bb9cd1c7563dd4) the problem where dragging gets messed up after window size is changed.
> 
> The fix is released in [v2.2.0](https://github.com/Temzasse/react-modal-sheet/releases/tag/v2.2.0). Please let me know if the problem still persists 😊

@Temzasse I have the exact same issue now, on version 4.4.0.

Exactly as shown in the OP's video.

This is my implementation of the sheet:

`
	<Sheet
		isOpen={opened}
		onClose={() => {
			snapTo(initialSnap);
		}}
		snapPoints={snapPoints}
		initialSnap={initialSnap}
		detent="content-height"
		disableScrollLocking
		ref={ref}
		style={{ zIndex: 50 }}
	>
		<Sheet.Container>
			<Sheet.Header /> 
			<Sheet.Content 
			ref={props.ref}
			>{children}</Sheet.Content>
		</Sheet.Container>
	</Sheet>
`

I'm not sure why this issue is happening again.
